test/EnvironmentManagerTests/Helper.h

```diff
@@ -0,0 +1,167 @@
+﻿#pragma once
+#include "pch.h"
+
+using namespace winrt::Microsoft::ProjectReunion; // for EnvironmentManager
+using namespace winrt::Windows::Foundation::Collections; // for IMapView
+typedef IMapView<winrt::hstring, winrt::hstring> EnvironmentVariables;
+
+inline const wchar_t* USER_EV_REG_LOCATION = L"Environment";
```

Comment:
Why `const wchar_t*`? Why not PCWSTR?

The latter includes a SAL annotation `_Null_terminated_`

Reply:
Because I didn't know PSWSTR was a thing.  I'll be changing them now.